index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+    <section class="grid">
+            <div>
+                <h2 class="headings">Projects</h2>
+            </div>
+     
+            <div class="project">
+                <div class="laptop-img">
+                    <img src="images/laptop.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
+                    <div class="details-button">
+                       <div>
+                        <p >See this project</p>
+                        <img src="images/Union.svg" alt="union">
+                       </div>
+                    </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
- [ ] Similarly, please make sure to use `display: flex` for individual project cards as mentioned in the project requirements ✨

  > _In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and **use Flexbox to lay out an individual card**._